airflow/hooks/hive_hooks.py

```diff
@@ -489,7 +489,8 @@ def to_csv(
             schema='default',
             delimiter=',',
             lineterminator='\r\n',
-            output_header=True):
+            output_header=True,
+            fetch_size=1000):
```

Comment:
Does this have the potential to break existing implementations of HiveHook, if someone is expecting > 1000 rows? I think this should default to `None`.